Cinema/src/main/java/com/example/Cinema/mapper/MoviesMapper.java

```diff
@@ -58,4 +58,8 @@
             "from movies " +
             "where id_genre = #{idGenre} ")
     List<Movies> getByGenre(int idGenre);
+
+    @Select("select name " +
```

Comment:
It would be a good idea to extract all query strings into a separate class for queries.

Reply:
Fixed